forc-pkg/src/manifest/mod.rs

```diff
@@ -206,15 +208,45 @@ pub struct Network {
     pub url: String,
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
+pub struct HexSalt(pub fuel_tx::Salt);
+
+impl FromStr for HexSalt {
+    type Err = anyhow::Error;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        // cut 0x from start.
+        let normalized = &s[2..];
```

Comment:
This will panic for strings smaller than 2 bytes, right?
You can safely do this with https://doc.rust-lang.org/std/primitive.str.html#method.strip_prefix

Reply:
That is a good catch! Thanks